### PR TITLE
Feat/generalize to ansys theme

### LINKED ISSUE #46: Generalize theme to become `ansys-sphinx-theme`
### Description of the feature

We should have a generalized sphinx theme that isn't just specific to pyansys. This was the major takeaway from our meeting with the various documentation/product teams.

### Steps for implementing the feature

- Change theme name to `ansys-sphinx-theme`
- Include Ansys logos, provide documentation for this.
- Change URL (later) . Include a redirect from sphinxdocs.pyansys.com. Probably will be something like sphinx.docs.ansys.com, but we'll see
- Update our sphinx theme logo to use the Ansys logo (with approval by marketing).

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Fix #46  Generalize the theme by Changing the theme name to ansys-sphinx-theme. Included ansys logos

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ jobs:
       - name: Upload wheel and sdist to artifacts
         uses: actions/upload-artifact@v2
         with:
-          name: pyasys-sphinx-theme
+          name: pyansys-sphinx-theme
```
Comment (akaszynski) [inline .github/workflows/ci_cd.yml:95]:
```suggestion
          name: ansys-sphinx-theme
```

File: doc/Makefile [hunk 2]
```diff
@@ -27,4 +27,4 @@ clean:
 pdf:
 	@$(SPHINXBUILD) -M latex "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	cd build/latex && latexmk -r latexmkrc -pdf *.tex -interaction=nonstopmode || true
-	(test -f build/latex/pyansys*.pdf && echo pdf exists) || exit 1
\ No newline at end of file
+	(test -f build/latex/ansys_sphinx_theme.pdf && echo pdf exists) || exit 1
```
Comment (akaszynski) [inline doc/Makefile:30]: There's no reason to remove `*`. 

```suggestion
	(test -f build/latex/ansys*.pdf && echo pdf exists) || exit 1
```

Comment (Revathyvenugopal162) [reply doc/Makefile:30]: @akaszynski For pdf cover we are using the images in the form of pdf, named ansys_logo_white.pdf and ansys_logo_white_cropped.pdf. so while building it was showing error.

File: doc/source/usage.rst [hunk 3]
```diff
@@ -58,13 +58,13 @@ Use your ``custom.css`` file to override the CSS style of this theme.
...
 ~~~~~~~~~~~~~~~~~~
-The PyAnsys Sphinx theme supports the display of breadcrumbs on
+The Ansys Sphinx theme supports the display of breadcrumbs on
 the body of documentation pages to make navigation easier. These 
 breadcrumbs are disabled by default. To add
 breadcrumbs to the pages of your documentation, in the ``conf.py``
 file, add ``"show_breadcrumbs": True`` to the ``html_theme_options`` dictionary.
 
-If you want to add additional 'root' breadcrumbs, such as to the PyAnsys
+If you want to add additional 'root' breadcrumbs, such as to the ansys
```
Comment (jorgepiloto) [inline doc/source/usage.rst:67]:
```suggestion
If you want to add additional 'root' breadcrumbs, such as to the Ansys
```

File: doc/source/usage.rst [hunk 4]
```diff
@@ -58,13 +58,13 @@ Use your ``custom.css`` file to override the CSS style of this theme.
 
 Adding Breadcrumbs
 ~~~~~~~~~~~~~~~~~~
-The PyAnsys Sphinx theme supports the display of breadcrumbs on
+The Ansys Sphinx theme supports the display of breadcrumbs on
```
Comment (jorgepiloto) [inline doc/source/usage.rst:61]:
```suggestion
The ``ansys-sphinx-theme`` supports the display of breadcrumbs on
```

File: src/ansys_sphinx_theme/theme.conf [hunk 5]
```diff
@@ -3,7 +3,7 @@ inherit = pydata_sphinx_theme
 pygments_style = friendly
 
 [options]
-github_url = https://github.com/pyansys
+github_url = https://github.com/ansys
 logo_link = https://docs.pyansys.com/
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/theme.conf:7]:
```suggestion
logo_link = https://www.ansys.com/
```

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
Just left some minor comments. This is almost ready to go. Thanks for addressing this @Revathyvenugopal162 🚀

File: README.rst [hunk 6]
```diff
@@ -1,5 +1,5 @@
-PyAnsys Sphinx Theme
-====================
+Ansys Sphinx Theme
+==================
 |pyansys| |python| |pypi| |GH-CI| |MIT| |black|
 
 .. |pyansys| image:: https://img.shields.io/badge/Py-Ansys-ffc107.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAABDklEQVQ4jWNgoDfg5mD8vE7q/3bpVyskbW0sMRUwofHD7Dh5OBkZGBgW7/3W2tZpa2tLQEOyOzeEsfumlK2tbVpaGj4N6jIs1lpsDAwMJ278sveMY2BgCA0NFRISwqkhyQ1q/Nyd3zg4OBgYGNjZ2ePi4rB5loGBhZnhxTLJ/9ulv26Q4uVk1NXV/f///////69du4Zdg78lx//t0v+3S88rFISInD59GqIH2esIJ8G9O2/XVwhjzpw5EAam1xkkBJn/bJX+v1365hxxuCAfH9+3b9/+////48cPuNehNsS7cDEzMTAwMMzb+Q2u4dOnT2vWrMHu9ZtzxP9vl/69RVpCkBlZ3N7enoDXBwEAAA+YYitOilMVAAAAAElFTkSuQmCC
```
Comment (jorgepiloto) [inline README.rst:5]:
```suggestion
.. |pyansys| image:: https://img.shields.io/badge/Py-Ansys-ffc107.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAIAAACQkWg2AAABDklEQVQ4jWNgoDfg5mD8vE7q/3bpVyskbW0sMRUwofHD7Dh5OBkZGBgW7/3W2tZpa2tLQEOyOzeEsfumlK2tbVpaGj4N6jIs1lpsDAwMJ278sveMY2BgCA0NFRISwqkhyQ1q/Nyd3zg4OBgYGNjZ2ePi4rB5loGBhZnhxTLJ/9ulv26Q4uVk1NXV/f///////69du4Zdg78lx//t0v+3S88rFISInD59GqIH2esIJ8G9O2/XVwhjzpw5EAam1xkkBJn/bJX+v1365hxxuCAfH9+3b9/+////48cPuNehNsS7cDEzMTAwMMzb+Q2u4dOnT2vWrMHu9ZtzxP9vl/69RVpCkBlZ3N7enoDXBwEAAA+YYitOilMVAAAAAElFTkSuQmCC
```

Comment (jorgepiloto) [reply README.rst:5]: Fix the `:target:` and `:alt:` attributes as well.

**Review by jorgepiloto — DISMISSED** [review]

**Review by akaszynski — APPROVED** [review]
LGTM

**Pull request merged**